src/NUnitFramework/framework/Api/FrameworkPackageSettings.cs

```diff
@@ -106,10 +106,15 @@ public static class FrameworkPackageSettings
         public const string DefaultTestNamePattern = "DefaultTestNamePattern";
 
         /// <summary>
-        /// Parameters to be passed on to the test
+        /// Parameters to be passed on to the tests, serialized to a single string which needs parsing. Obsoleted by <see cref="TestParametersDictionary"/>; kept for backward compatibility.
         /// </summary>
         public const string TestParameters = "TestParameters";
 
+        /// <summary>
+        /// Parameters to be passed on to the tests, already parsed into an IDictionary&lt;string, string>. Replaces <see cref="TestParameters"/>.
```

Comment:
I would have done the whole `<see cref="IDictionary{X,Y}"/>` but there is no facility for specifying closed generics, so users would have seen `IDictionary<TKey, TValue>` no matter what I wrote for `X` and `Y`. :(